BrowserKit/Sources/ComponentLibrary/DefaultTextField.swift

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Common
+import UIKit
+
+public class DefaultTextField: UITextField, ThemeApplicable {
```

Comment:
thanks for adding a new component to the library! Could we add this to the sample component library as well as update the docs for this new reusable component? Can be done in a separate PR, but wanted to make sure we are tracking this.

Reply:
Yes Cyndi. i'll add a task to create docs for it and add it to the sample app